west.yml

```diff
@@ -141,7 +141,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: f8ceac162de19cbcee9652cb428b217b1cf85ab3
+      revision: pull/1089/head
```

Comment:
`6db09621d01fd246d21010adc6ac8bdcb2a0e16a`

Reply:
Thank you